src/index.tsx

```diff
@@ -41,6 +41,16 @@ const styles = StyleSheet.create({
 
 export const SafeAreaConsumer = SafeAreaContext.Consumer;
 
+export function withSafeAreaConsumer<T>(
```

Comment:
```suggestion
export function withSafeAreaInsets<T>(
```

Reply:
perfect, let me do that!

Reply:
Done! please let me know what you think!